multisrc/overrides/wpmangastream/mareceh/src/Mareceh.kt

```diff
@@ -0,0 +1,21 @@
+package eu.kanade.tachiyomi.extension.id.mareceh
```

Comment:
Remember to change the package inside the class too.

```suggestion
package eu.kanade.tachiyomi.extension.id.mangceh
```

Reply:
Roger